### PR TITLE
Added LintLizard

### PR DESCRIPTION
**Pull request by svenkapudija-close**
Added LintLizard and ran the 1st global reformat before working on other branches/improvements.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    #B011
+exclude=venv
+#max-complexity=2
+banned-modules=
+    typing.Text = use str
+require-code=True
+
+[isort]
+skip=venv
+known_first_party=lintlizard
```
Comment (tsx) [inline setup.cfg:52]: Probably want to customize this

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+[mypy]
+python_version = 3.6
+ignore_missing_imports = True
+no_implicit_optional = True
+strict_equality = True
+follow_imports = skip
+warn_unreachable = True
+show_error_context = True
+pretty = True
+files = lintlizard
```
Comment (tsx) [inline setup.cfg:73]: And this

**Review by eengoron — APPROVED** [review]
Other than V's comments LGTM!

**Comment by svenkapudija-close** [comment]
@tsx Removed `lintlizard` as a 1st party, and changed mypy to `scripts` path (even though we don't use any at this time).

**Review by tsx — APPROVED** [review]

**Pull request merged**